generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -205,6 +205,24 @@ class SmallSPIFlashRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithCoherentBusTopology ++
   new freechips.rocketchip.system.BaseConfig)
 
+class SimAXIRocketConfig extends Config(
+  new chipyard.iobinders.WithUARTAdapter ++
+  new chipyard.iobinders.WithTieOffInterrupts ++
+  new chipyard.iobinders.WithSimAXIMem ++                       // drive the master AXI4 memory with a SimAXIMem
+  new chipyard.iobinders.WithTiedOffDebug ++
```

Comment:
Is there a strong motivation for anyone to use this config, instead of the BlackBoxSimMem?

Reply:
No idea. I needed to test my changes so I made a config that used it, and thought it would be easier to include it than not. We can remove it if its not useful.

Reply:
I think it is fine to have a config.